kateryna-tolochko/README.md

```diff
@@ -0,0 +1,97 @@
+# Getting Started with AliExpress
+
+AliExpress is a popular online marketplace where you can find a wide variety of products at affordable prices. Whatever you are looking for, AliExpress offers a huge selection from sellers around the world. If you are new to AliExpress, this guide will help you get started and make the most of your shopping experience.
+
+## Step 1: Create an Account
+
+1. Visit the [AliExpress website](https://www.aliexpress.com/).
+
+2. Find **Join Free** button and click it.
+
+   ![ ](https://i.ibb.co/cyYxdZX/screen.png)
```

Comment:
It's a good practice to add image text in the brackets to describe the linked image https://learn.microsoft.com/en-us/azure/devops/project/wiki/markdown-guidance?view=azure-devops#images. It might be useful for users with disabilities.